### ISSUE TITLE
WATISRAIN

### ISSUE THREAD

**Issue by viordano** [luckytoilet/WATisRain#43]
Hey, I'm not sure if you have it for ios or not.. but it seems like you do and whenever i click to download it it says it isn't available in Canada.

**Comment by luckytoilet** [comment]
Yes, the iOS version exists but is discontinued because I can't afford to pay the annual fee to keep it on the app store. If you like, you can build it from source.

**Comment by viordano** [comment]
What do you mean by build it from the source? I’m not familiar with programming, is there anything I can do?

**Comment by luckytoilet** [comment]
Ah nevermind then. The app is no longer available on iOS.
